client/src/components/HandMarkedPaperBallot.tsx

```diff
@@ -261,23 +261,17 @@ const HandMarkedPaperBallot = ({
   useLayoutEffect(() => {
     const printBallot = printBallotRef?.current
 
-    if (!printBallot) {
-      return
-    }
-
-    ; (async () => {
-      const flow = await new Previewer().preview(
-        ballotRef.current!.innerHTML,
-        ['/ballot/ballot.css'],
-        printBallot
-      )
-      console.log('preview rendered, total pages', flow.total, { flow })
-      onRendered?.({ ballotStyleId, election, isLiveMode, precinctId, votes })
-    })()
+    new Previewer().preview(
+      ballotRef.current!.innerHTML,
+      ['/ballot/ballot.css'],
+      printBallot
+    )
+    onRendered?.({ ballotStyleId, election, isLiveMode, precinctId, votes })
 
     return () => {
-      // console.log('removing pagedjs ballot')
-      printBallot.innerHTML = ''
+      if (!!printBallot) {
+        printBallot.innerHTML = ''
+      }
```

Comment:
```suggestion
      printBallot?.innerHTML = ''
```

Reply:
![image](https://user-images.githubusercontent.com/28444/84448829-951e7480-ac00-11ea-8182-b5bc01aef4ed.png)

Reply:
I also tried `printBallot && printBallot.innerHTML = ''` but then got a warning that there was a missing `;` somewhere 🤷

Reply:
Ah, right. Well at least you can just do `if (printBallot) {` rather than `!!`.